src/pages/App.tsx

```diff
@@ -76,35 +77,43 @@ export default function App() {
   }, [])
 
   return (
-    <Suspense fallback={null}>
-      <SkeletonTheme baseColor={theme.bg3} highlightColor={theme.bg2}>
-        <ApolloProvider client={subgraphClients[chainId as SWPRSupportedChains] || defaultSubgraphClient}>
-          <NetworkWarningModal />
-          <AppWrapper id="app-wrapper">
-            <HeaderWrapper>
-              <Header />
-            </HeaderWrapper>
-            <BodyWrapper>
-              <Web3ReactManager>
-                <SpaceBg>
-                  <Suspense fallback={<FallbackLoader />}>
-                    <Routes />
-                  </Suspense>
-                </SpaceBg>
-              </Web3ReactManager>
-              <Marginer />
-            </BodyWrapper>
-          </AppWrapper>
-          <ToastContainer
-            draggable={false}
-            className="custom-toast-root"
-            toastClassName="custom-toast-container"
-            bodyClassName="custom-toast-body"
-            position="top-right"
-            transition={Slide}
-          />
-        </ApolloProvider>
-      </SkeletonTheme>
-    </Suspense>
+    <>
+      {/* All pages contain the same meta, move it to proper pages when content is ready */}
+      <PageMetaData
+        title="Swap"
+        description="A governance-enabled automated market maker with adjustable fees."
+        noIndex={process.env.REACT_APP_BUILD_ENV !== 'production'}
```

Comment:
```suggestion
        noIndex={process.env.REACT_APP_BUILD_ENV === 'staging'}
```
This` !== 'production'  ` does not work. Since we have this environment variable `REACT_APP_BUILD_ENV` only in netlify. And the value is `staging` all other places this varibale has value as `undefined` 

So all our `netlify PR builds` and `swapr.site` is in netlify so we can check 
process.env.REACT_APP_BUILD_ENV === 'staging'

In local development anyway its not an issue of SEO.

We deploy through IPFS so I am not sure how to configure there. For our usecase its best to check only in netlify

Reply:
Yes you need to check == 'staging'